docs/pages/docs/guide/advanced/table.mdx

```diff
@@ -158,9 +160,9 @@ components `<tr />{:jsx}`, `<th />{:jsx}` and `<td />{:jsx}`, for the same
 reason `<table />{:jsx}` literal is not replaced and doesn't have default
 margin-top aka `mt-6`.
 
-### Ways to Fix It
+## Ways to Fix It
 
-#### One-Time Fix
+### One-Time Fix
```

Comment:
I rollback this heading since a section with `remark-mdx-disable-explicit-jsx` works, maybe you have idea of better name?